docs/book/src/cronjob-tutorial/testdata/project/api/v1/cronjob_webhook.go

```diff
@@ -65,26 +70,42 @@ A webhook will automatically be served that calls this defaulting.
 The `Default` method is expected to mutate the receiver, setting the defaults.
 */
 
-var _ webhook.Defaulter = &CronJob{}
+type CronJobCustomDefaulter struct{}
 
-// Default implements webhook.Defaulter so a webhook will be registered for the type
-func (r *CronJob) Default() {
-	cronjoblog.Info("default", "name", r.Name)
+var _ webhook.CustomDefaulter = &CronJobCustomDefaulter{}
 
-	if r.Spec.ConcurrencyPolicy == "" {
-		r.Spec.ConcurrencyPolicy = AllowConcurrent
+// Default implements webhook.CustomDefaulter so a webhook will be registered for the type
+func (d *CronJobCustomDefaulter) Default(ctx context.Context, obj runtime.Object) error {
+	cronjoblog.Info("CustomDefaulter for Admiral")
```

Comment:
```suggestion
	cronjoblog.Info("CustomDefaulter for CronJob")
```
Is this supposed to be CronJob?  The validator below it shows CronJob